src/test/java/org/lookout_studios/meals_management_system/meals_management_system/MealsManagementSystemApplicationTests.java

```diff
@@ -8,6 +8,8 @@ class MealsManagementSystemApplicationTests {
 
 	@Test
 	void contextLoads() {
+		EstablishConnectionWithDatabase mysqlConnectionObject = new EstablishConnectionWithDatabase () ;
+		mysqlConnectionObject.createSqlConnection ();
```

Comment:
This test does not make any assertions and as such doesn't test anything. Check several test cases. Mock the database behavior if you have to.